Judger/src/main/java/tw/waterball/judgegirl/judger/CCJudger.java

```diff
@@ -190,20 +190,25 @@ protected void onBeforeRunningTestcase(Testcase testcase) {
           (The files-difference must be 'std.out', 'std.err' and out-files)
          */
         filesWithinSandboxRootOtherThanOutFiles =
-                generateFilesOtherThanOutFilesFromSandboxRoot(sandboxRootPath);
+                generateFilesOtherThanOutFilesFromSandboxRoot(sandboxRootPath, testcase);
 
         copyExecutableIntoSandboxRoot(testcase);
         // TODO should also copy provided codes, this will fail in an interpreted language case
         copyInterpretedSubmittedCodesIntoSandboxRoot(testcase);
     }
 
     @NotNull
-    private Set<String> generateFilesOtherThanOutFilesFromSandboxRoot(Path sandboxRootPath) {
+    private Set<String> generateFilesOtherThanOutFilesFromSandboxRoot(Path sandboxRootPath, Testcase testcase) {
         var files = stream(requireNonNull(sandboxRootPath.toFile().listFiles()))
                 .map(File::getName).collect(Collectors.toSet());
         files.add("std.out");
         files.add("std.err");
         files.add(EXECUTABLE_NAME);
+        try {
+            FileUtils.touch(FileUtils.getFile(getTestcaseOutputHome(testcase).getPath().toString(), "std.out"));
+        } catch (Exception exception) {
+            throw new RuntimeException();
+        }
```

Comment:
1. I think we don't need the utils, just write ` Files.createFile(getTestcaseOutputHome(testcase).getPath().resolve("std.out"));`

2. don't do try-catch here, annotating the method with `@SneakyThrows` does this for you. 

3. Don't modify the method `generateFilesOtherThanOutFilesFromSandboxRoot`, I think put this in `onBeforeRunningTestcase` above the variable `filesWithinSandboxRootOtherThanOutFiles` is well.